docs/en/revocation-lists.rst

```diff
@@ -100,6 +100,9 @@ Credential Revocation Flows can start under different scenarios, such as:
     - The Authentic Sources that for any administrative reasons update one or more attributes of a User, shall inform the Issuer of related Credentials. 
     - The Issuers, for technical security reasons (e.g. in the case of compromised cryptographic keys, death of the User, etc.), can decide to revoke the Credentials.
 
+The Credential in some cases MUST be considered valid (thus is still accurate, useful and valid) for the period before the revocation: a driving license, revoked due expiration or other reasons, is still valid for the period before the revocation.
+ 
+Revocation MUST therefore include a flag indicating whether it is retroactive or not. It SHOULD not invalidate further use of the credential, as it MAY be used to attest activity outside the revocation interval. If the "retroactive flag" is activated, the credential MUST be invalid  also for the period prior to the revocation date. If the "retroactive flag" is not activated, the credential MUST be considered valid up to revocation date and could be used to demonstrate that, before revocation, the holder was entitled to perform related activities.
```

Comment:
Where this "flag" was discussed?

Reply:
When there are 2 only possible options, you can differentiate them with a flag instead of having different parameters as the differentiating factor doesn't change the overall specifications. This is an approach, if you prefer different approach let us know

Reply:
I would be cautious not to expose/mix operational guidance to issuers and verifiers, with protocol guidance to implementers.

For example:

It is recommended that Issuers maintain a log including the time and optionally reason for revocation or suspensions.

Retention policies for such logs are out  of scope for this specification, and may be constrained by regional requirements, etc...